scripts/runalltests.sh

```diff
@@ -0,0 +1,37 @@
+workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"/testslog
+testdir=$workdir/../../../tests
+
+if [ -d $workdir ]; then
+rm -r $workdir
+fi
+mkdir $workdir
+exec &> $workdir/testlog.txt
+cd $testdir
+
+echo "Fetching the test repository:"
+git pull
+echo "Test Repository last commit hash:"
+git rev-parse HEAD
+cd ..
```

Comment:
so you suggest to clone the repos on a fly? 
but keep this script as a part of cpp-ethereum. 
as it is now script should update itself by calling git pull